ingestion_tools/scripts/common/mesh_converter.py

```diff
@@ -0,0 +1,106 @@
+import logging
+import typing
+
+import numpy as np
+import trimesh
+
+logger = logging.getLogger(__name__)
+
+
+def convert_mesh_to_glb(convert_func: typing.Callable[[str], trimesh.Trimesh]) -> typing.Callable:
+    """
+    A decorator to convert and scale a mesh by a given factor
+
+    :param convert_func: the function to convert a mesh file to a trimesh object.
+    """
+
+    def wrapper(input_file: str, output_file: str, scale_factor: float = 1.0) -> None:
```

Comment:
As the mesh is 3d, should the scale factor be a list of 3 values?

Reply:
It doesn't have to be if everything is scaled by the same amount for each direction. Is that something we need to support?

Reply:
@uermel what do you think?

Reply:
One scale factor is enough. I don't know of any case where anisotropic scaling would be applied in regular cryoET processing. We also don't do that for points.